fairytale.md

```diff
@@ -44,3 +44,10 @@ So, he also boiled dumplings, and together with his son Peter,  tasted the dumpl
 
 `Well... your turn now.`
 
+
+Chop Chop and Samsung headphones dont go together!!
```

Comment:
Hi, this content is not accepted as is. Please move it above the straight line (line 36 in this pull request) but I suggest you pull from my upstream main before doing that because there has been a lot of changes including a message about content not accepted below the line.